updater/aws.go

```diff
@@ -398,6 +398,9 @@ func (u *updater) getCommandResult(commandID string, instanceID string) ([]byte,
 		return nil, fmt.Errorf("failed to retrieve command invocation output: %w", err)
 	}
 	commandResults := []byte(aws.StringValue(resp.StandardOutputContent))
+	if aws.StringValue(resp.Status) != "Success" {
+		return nil, fmt.Errorf("commandID %s failed: %s", commandID, aws.StringValue(resp.StatusDetails))
```

Comment:
// I think this might make it more clear why it failed.
```suggestion
		return nil, fmt.Errorf("command %s  has not reached success status, current status %q", commandID, aws.StringValue(resp.Status))
```